src/test/java/com/github/alexcojocaru/mojo/elasticsearch/v2/step/ValidateClusterNameStepTest.java

```diff
@@ -0,0 +1,42 @@
+package com.github.alexcojocaru.mojo.elasticsearch.v2.step;
+
+import com.github.alexcojocaru.mojo.elasticsearch.v2.ClusterConfiguration;
+import com.github.alexcojocaru.mojo.elasticsearch.v2.ElasticsearchSetupException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ValidateClusterNameStepTest {
+    private ValidateClusterNameStep validateClusterNameStep;
+    ClusterConfiguration.Builder configurationBuilder;
+
+    @Before
+    public void setUp() {
+        validateClusterNameStep = new ValidateClusterNameStep();
+        configurationBuilder = new ClusterConfiguration.Builder();
+    }
+
+    @After
+    public void tearDown() {
+        validateClusterNameStep = null;
+        configurationBuilder = null;
+    }
+
+    @Test
+    public void TestWithClusterNameContainingDash() {
+        ClusterConfiguration clusterConfiguration = configurationBuilder.withClusterName("ONE-TWO-THREE").build();

```

Comment:
keep the lines under 100 chars long please, except for those which contain a single string

Reply:
All integration tests have been failing, I believe it's because of the change in ForkedInstance.java. On first glance, it looks like the instance is starting before each integration test, but its name is not the one provided via the arguments, and hence the monitor cannot connect to it and it fails after 30 secs, thus failing the integration test.